addons/im_livechat/static/src/legacy/public_livechat.xml

```diff
@@ -17,7 +17,7 @@
             <div class="o_livechat_rating_reason">
                 <textarea id="reason" placeholder="Explain your note"></textarea>
                 <div class="o_livechat_rating_reason_button">
-                    <input type="button" class="btn btn-primary btn-sm o_rating_submit_button" value="Send" />
+                    <button type="button" class="btn btn-primary btn-sm o_rating_submit_button">Send</button>
```

Comment:
What's the reason to change `input` from `button`?
It doesn't seem to be related to the issue at hand that it's not translated properly. It looks more a style fix that is not documented in commit message/title. Do you confirm this is the reasoning of change?

Reply:
Oh right, makes sense then :)